test/FilesTests/RavenFilesTestBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
I think it is better to be isolated, so if you work on FS you can run just those tests and if you word of documents you don't care about those tests.
But if you do not agree, I will merge them now.